core/debugger/remote_debugger.cpp

```diff
@@ -408,12 +408,10 @@ void RemoteDebugger::debug(bool p_can_continue, bool p_is_error_breakpoint) {
 	}
 
 	ScriptLanguage *script_lang = script_debugger->get_break_language();
-	ERR_FAIL_NULL(script_lang);
-
 	Array msg = {
 		p_can_continue,
-		script_lang->debug_get_error(),
-		script_lang->debug_get_stack_level_count() > 0,
+		script_lang ? script_lang->debug_get_error() : String(),
+		script_lang ? script_lang->debug_get_stack_level_count() > 0 : false,
```

Comment:
```suggestion
		script_lang && (script_lang->debug_get_stack_level_count() > 0),
```
No need for a ternary for booleans